examples/migration/campaign_management/create_complete_campaign_adwords_api_only.rb

```diff
@@ -0,0 +1,234 @@
+# Encoding: utf-8
+#
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This code example is the first in a series of code examples that shows how to create
+# a Search campaign using the AdWords API, and then migrate it to Google Ads API one
+# functionality at a time. See other examples in this directory for code examples in various
+# stages of migration.
+#
+# This code example represents the initial state, where the AdWords API is used to create a
+# campaign budget, a Search campaign, ad groups, keywords and expanded text ads. None of the
+# functionality has yet been migrated to the Google Ads API.
+
+
+NUMBER_OF_ADS = 5
+KEYWORDS_TO_ADD = ["mars cruise", "space hotel"]
+API_VERSION = "201809"
```

Comment:
You're specifying this in 2 places. Constants like this should be specified at the bottom of the file by convention in our examples, including the other constants you have in this section.